templates/java/org/prism/Loader.java.erb

```diff
@@ -310,6 +310,18 @@ public class Loader {
         return (short) flags;
     }
 
+    private VariableInteger loadVariableInteger() {
+        boolean negative = buffer.get() != 0;
+        int length = loadVarUInt();
+        int[] nodes = new int[length];
```

Comment:
nodes -> parts or so

Reply:
In [rb_integer_pack](https://github.com/ruby/ruby/blob/e04146129ec6898dd6a9739dad2983c6e9b68056/include/ruby/internal/intern/bignum.h#L579-L679) terms that's called `words`.